### PR TITLE
recursively render up to 3 layers of placeholder tiles

### PR DESCRIPTION
**Pull request by veggiesaurus**
This solves #363 by recursively calling `renderTiles` (up to 3 times) for higher resolution tiles (it's already done for lower-resolution tiles). 

The higher-resolution tiles are attempted first, since by writing to the z-buffer, they'll appear on top of any lower-resolution tiles.

### CODE REVIEW

File: src/services/BackendService.ts [hunk 1]
```diff
@@ -65,7 +65,6 @@ export class BackendService {
             CARTA.EventType.OPEN_FILE,
             CARTA.EventType.OPEN_FILE_ACK,
             CARTA.EventType.RASTER_IMAGE_DATA,
-            CARTA.EventType.REGION_HISTOGRAM_DATA
```
Comment (duidae) [inline src/services/BackendService.ts]: sorry this part is deleted due to the log event PR

Comment (veggiesaurus) [reply src/services/BackendService.ts]: solving merge conflics now

**Comment by veggiesaurus** [comment]
@duidae merge commits resolved, please review :)

**Comment by duidae** [comment]
@veggiesaurus thanks!

**Pull request merged**